pkg/storaged/lvol-tabs.jsx

```diff
@@ -110,6 +114,15 @@
                 return Promise.reject(_("VDO backing devices can not be made smaller"));
             else
                 return Promise.resolve();
+        } else if (stratis_bdev) {
+            const delta = size - crypto_overhead - Number(stratis_bdev.TotalPhysicalSize);
+            if (delta > 0) {
+                const pool = client.stratis_pools[stratis_bdev.Pool];
+                return pool.GrowPhysicalDevice(stratis_bdev.Uuid).then(std_reply);
+            } else if (delta < 0)
+                return Promise.reject(_("Stratis blockdevs can not be made smaller"));
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19079-20230825-132955-2df3b918-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)